pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2182,8 +2182,8 @@ protected void handleGetTopicsOfNamespace(CommandGetTopicsOfNamespace commandGet
                     getBrokerService().pulsar().getNamespaceService().getListOfTopics(namespaceName, mode)
                         .thenAccept(topics -> {
                             boolean filterTopics = false;
-                            // filter transaction internal topic
-                            List<String> filteredTopics = TopicList.filterTransactionInternalName(topics);
+                            // filter system topic
+                            List<String> filteredTopics = TopicList.filterSystemTopic(topics);
```

Comment:
> Why? I think we should filter all system topics, these topics should be invisible to the user.

The original implementation only filters out topics of type TXN, system topics have other type topics, which would be a break change

Reply:
I think we should discuss clearly first: What system topic should be exposed to users?

These system themes are all used internally(`__change_event` and transaction-related topics), and I don't think they should be exposed to users.

So, Maybe this PR isn't a breaking change, It fixes compatibility issues caused by PRs like #16042

Reply:
> These system themes are all used internally(__change_event and transaction-related topics), and I don't think they should be exposed to users.

I think so too. But before we filter out all the system topics, should we send an email to discuss it first?

Reply:
Sorry, I forgot to post the discussion mail: https://lists.apache.org/thread/qhm0zbfw0wfd1m5vj977w002qmkr5lt9